numenc-cpp/encoder_decoder.cpp

```diff
@@ -345,29 +342,28 @@ static PyObject * to_int32(PyObject * self, PyObject * args) {
     return output;
 }
 
-static PyObject * from_uint32(PyObject * self, PyObject * args) {
-    long input;
-    PyObject * output;
+static PyObject* from_uint32(PyObject* self, PyObject* args) {
+    long long input;
```

Comment:
Since python is using big (unbounded) integers, please double-check that you test with 128-bit or 256-bit integers which burst the size of `long long`.